packages/aws-cdk-lib/aws-rds/lib/instance.ts

```diff
@@ -713,6 +754,20 @@ export interface DatabaseInstanceNewProps {
    * @default - IPV4
    */
   readonly networkType?: NetworkType;
+
+  /**
+   * The identifier of the CA certificate for this DB instance.
+   *
+   * Specifying or updating this property triggers a reboot.
+   *
+   * For RDS DB engines:
+   * @see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html
+   * For Aurora DB engines:
+   * @see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.SSL-certificate-rotation.html
+   *
+   * @default - CaCertificate.RDS_CA_2019
```

Comment:
This default is unexpected and seems like a potentially breaking change to me. Can you explain or remove it?

Reply:
Trying to create an instance from the console, it suggests `rds-ca-2019` as default (this is the reason for the `@default`).

<img width="748" alt="Screenshot 2023-08-28 at 15 53 43" src="https://github.com/aws/aws-cdk/assets/15077781/244cd1af-df55-42f9-82d3-0c8e590eeef5">

Anyway, looking at the CFN [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html#cfn-rds-dbinstance-cacertificateidentifier), it does not specify any default, so I guess I should remove it.

Reply:
I see. It probably doesn't change much, but I think we want to end up in this case:
"If you don't select a certificate authority, RDS chooses one for you."

```suggestion
   * @default - Let RDS chooses a certificate authority for you
```